src/index.html

```diff
@@ -1,12 +1,311 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./styles/main.scss" />
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Catalog</h1>
+    <div class="conteiner">
+      <!-- #region header -->
+      <header class="main">
+        <div class="main-content">
+          <a href="#" class="main-content__logo">
+            <img src="./images/logo.png" alt="img" />
+          </a>
+
+          <nav class="main-content__nav">
+            <ul class="main-content__list">
+              <li class="main-content__item">
+                <a href="/" class="main-content__link is-active"> Apple </a>
+              </li>
+              <li class="main-content__item">
+                <a href="/" class="main-content__link"> Samsung </a>
+              </li>
+              <li class="main-content__item">
+                <a href="/" class="main-content__link"> Smartphones </a>
+              </li>
+              <li class="main-content__item" data-qa="nav-hover">
+                <a href="/" class="main-content__link"> Laptops & Computers </a>
+              </li>
+              <li class="main-content__item">
+                <a href="/" class="main-content__link"> Gadgets </a>
+              </li>
+              <li class="main-content__item">
+                <a href="/" class="main-content__link"> Tablets </a>
+              </li>
+              <li class="main-content__item">
+                <a href="/" class="main-content__link"> Photo </a>
+              </li>
+              <li class="main-content__item">
+                <a href="/" class="main-content__link"> Video </a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </header>
+      <!-- #endregion -->
+
+      <!-- #region card -->
+      <main class="cards">
+        <div class="card" data-qa="card">
+          <img src="./images/imac.jpeg" alt="Apple iMac" />
+
+          <div class="card__details">
+            <h2 class="card__details-title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h2>
+            <p class="card__details-code">Product code: 195434</p>
+          </div>
+
+          <div class="card__stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__stars-review">Reviews: 5</p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-text">Price:</p>
+            <p class="card__price-number">$2,199</p>
+          </div>
+
+          <div class="card__button">
```

Comment:
This div is redundant
```suggestion
```

Reply:
To this element most of the styles for the button are connected.
Is there a better solution for this? Wouldbe happy to hear suggestions  :)

Reply:
Thanks for the feedback, I've added the element around "Buy" because I'm not able to add more than 2 attributes. Nothing can be removed, I needed class for styling,